earthquakes/notebooks/source memory.ipynb

```diff
@@ -386,6 +391,39 @@
    "source": [
     "submission.to_csv(os.path.join(data_dir, \"submissions\", \"submission_full_cycles_XGBoost_extened_dataset.csv\"), index=False)"
    ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## 4. Compare distributions\n",
+    "Here we check if the data distributions of the features on the train and test are the same Kolmogorov-Smirnov test.\n"
+   ]
```

Comment:
I checked for some distributions of random features and found that the results are not always what you expect. For example, check the `0_mean` in this figure, which are clearly different distributions, yet the function says "same dist".

![image](https://user-images.githubusercontent.com/32506895/55288462-63810b00-53b8-11e9-9606-c71da06f23b8.png)

Maybe we should use a higher default value of alpha? Then we will reject the null-hypothesis more often, which is the more conservative options in our case.

Reply:
The problem was that I was testing train and train set. Hence, always the same distribution. I corrected it at helpers.